owncloudApp/src/main/java/com/owncloud/android/shares/ui/fragment/ShareFileFragment.kt

```diff
@@ -479,7 +514,7 @@ class ShareFileFragment : Fragment(), ShareUserListAdapter.ShareUserAdapterListe
         )
 
         // Show or hide public links and no public links message
-        if (publicLinks!!.size > 0) {
+        if (publicLinks?.size!! > 0) {
```

Comment:
Is` !! `necessary? Because you use it later without `!!`

Reply:
Same as https://github.com/owncloud/android/pull/2508#discussion_r279286131